src/cup1d/likelihood/lya_theory.py

```diff
@@ -28,351 +42,503 @@ def __init__(self,zs,emulator=None,verbose=False,
             - P_model_fid: fiducial pressure model
             - include_metals: list of metal labels to include
             - cosmo_fid: fiducial cosmology used for fixed parameters
+            - sim_igm: specify file with fiducial IGM models
+            - emu_type: type of emulator: gp or nn
         """
 
-        self.verbose=verbose
-        self.zs=zs
-        self.emulator=emulator
+        self.verbose = verbose
+        self.zs = zs
+
+        # specify pivot point used in compressed parameters
+        self.z_star = z_star
+        self.kp_kms = kp_kms
+
+        # setup emulator
+        self.emulator = emulator
+        if emu_type == "nn":
+            self.get_emulator_calls = self.get_emulator_calls_nn
+            self.call_emulator = self.emulator.emulate_arr_p1d_Mpc
+        elif emu_type == "gp":
+            self.get_emulator_calls = self.get_emulator_calls_gp
+            self.call_emulator = self.emulator.emulate_p1d_Mpc
+        else:
+            raise ValueError("Only nn and gp emu_type implemented")
 
         # specify pivot point used in emulator
         if self.emulator is None:
-            print('using default values for emulator pivot point')
-            self.emu_kp_Mpc=0.7
+            print("using default values for emulator pivot point")
+            self.emu_kp_Mpc = 0.7
         else:
-            self.emu_kp_Mpc=self.emulator.archive.kp_Mpc
-
-        # specify pivot point used in compressed parameters
-        self.z_star=3.0
-        self.kp_kms=0.009
+            self.emu_kp_Mpc = self.emulator.archive.kp_Mpc
+
+        # load IGM history
```

Comment:
I would clarify here again that this is about the fiducial IGM history to be used, not necessarily the true one. 

I'm saying this in particular because we will also need one when running on real data, and the current code might seem that it can only be used with simulated data

Reply:
Done!